pkg/client/didexchange/client.go

```diff
@@ -94,7 +96,7 @@ func New(ctx provider) (*Client, error) {
 		didexchangeSvc:           didexchangeSvc,
 		kms:                      ctx.KMS(),
 		inboundTransportEndpoint: ctx.InboundTransportEndpoint(),
-		connectionStore:          didexchange.NewConnectionRecorder(transientStore, store),
+		connectionStore:          didexchange.NewConnectionRecorder(transientStore, store, ctx.DIDConnectionStore()),
```

Comment:
we don't need to pass DIDConnectionStore. This issue will be fixed #999

Reply:
it is not my code :) I pulled @Moopli changes

Reply:
this connection store is different from the one that was in didExchange - this maps keys to DIDs, and theirKeys to myKeys.